nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -216,20 +216,26 @@ in
       debug = mkEnableOption "gnome-session debug messages";
 
       flashback = {
-        enableMetacity = mkEnableOption "the standard GNOME Flashback session with Metacity";
+        enableMetacity = mkOption {
+          type = types.bool;
+          default = true;
+          example = "false";
+          description = "Whether to enable the standard Metacity GNOME flashback session.";
+        };
 
         customSessions = mkOption {
           type = types.listOf (types.submodule {
             options = {
+              # Deprecated, kept here just to show an error later.
```

Comment:
Apparently, it is just possible to pass `imports` attribute with `mkRemovedOptionModule` `types.submodule`.

Reply:
Sorry, I don't really get this. Could you give an example?

Reply:
It would be something like:

```patch
+            imports = [
+              (mkRemovedOptionModule "wmName" "The session name is now automatically generated.")
+            ];
+
             options = {
-              # Deprecated, kept here just to show an error later.
```

But I still think human-readable names would be more convenient for debugging.

Reply:
I get:
```
The option `services.xserver.desktopManager.gnome.flashback.customSessions.*.assertions' does not exist. Definition values:
- In `/etc/nixos/nixpkgs/nixos/modules/services/x11/desktop-managers/gnome.nix':
    [
      {
        assertion = true;
        message = ''
          The option definition `wmName' in  no longer has any effect; please remove it.
    ...
```

Reply:
Oh, looks like the [PR allowing that](https://github.com/NixOS/nixpkgs/pull/97023) has been reverted :crying_cat_face: